Source/WebCore/rendering/RenderFragmentContainer.cpp

```diff
@@ -373,9 +374,7 @@ void RenderFragmentContainer::computeIntrinsicLogicalWidths(LayoutUnit& minLogic
         RenderBlockFlow::computeIntrinsicLogicalWidths(minLogicalWidth, maxLogicalWidth);
         return;
     }
-
-    minLogicalWidth = m_fragmentedFlow->minPreferredLogicalWidth();
-    maxLogicalWidth = m_fragmentedFlow->maxPreferredLogicalWidth();
+    maxLogicalWidth = minLogicalWidth = LayoutUnit();
```

Comment:
WebKit's style is as follows:
maxLogicalWidth = { }; 
minLogicalWidth = { };

Reply:
Cool! Will push updated commit once back (out right now). 👍

Reply:
Done.